### PR TITLE
added "Changing Release-Channels" blog

### LINKED ISSUE #1207: Publish "Changing Release-Channels" blog
The blog on [Changing Meshery Release channels](https://layer5.io/blog/meshery/changing-meshery-release-channels) has been published on [Layer5.io](https://layer5.io/) website, we need the same blog to be published on [meshery.io/blogs](https://meshery.io/blog) too

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and designs for Meshery Site in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Mohith234**
**Description**

This PR fixes #1207 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@Mohith234
- Make sure you are using relevant tags for code blocks
example: [meshery.io/#getting-started](https://meshery.io/#getting-started)
- Keep the thumbnail on the top, similar to [layer5.io/blog](https://meshery.io/#getting-started)
- Review and fix the formatting bugs

**Comment by Mohith234** [comment]
Hey @thisiskaransgit , I have some followup questions 
- Changing the thumbnail to the top disturbs the uniformity design for the list of blogs in this page https://meshery.io/blog. Is it ok still ?
- Also I've tried to incorporate the tags from here [meshery.io/#getting-started](meshery.io/#getting-started) But there it was a html doc and it got rendered as a code. Here it is not rendering a code block when `<code>` tag is used. Preview of that file in VS Code works fine but the output on the site doesn't match.

**Comment by vishalvivekm** [comment]
@Mohith234 Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Comment by thisiskaransgit** [comment]
@Mohith234, use the raw html elements in the markdown file for `copy to clipboard` 
```html
<div id="div-id" class="highlight-code">
  <code class="code-box">mesheryctl system context create docker --platform docker --set
    mesheryctl system start</code>
    <!-- copy to clipboard -->
    <a class="btn tooltip" style="position: absolute; top:0.10rem; right: 0.5rem; font-size: 22px;" data-clipboard-target="#div-id"
      data-clipboard-text="mesheryctl system context create docker --platform docker --set
      mesheryctl system start" onmouseout="resetCopyText(this)">
      <i class="far fa-copy"></i>
      <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 2rem; left: -80px;">Copy to clipboard</span>
    </a>
  </div>
```

**Comment by vishalvivekm** [comment]
@Mohith234 Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

File: collections/_posts/2023-06-16-changing-meshery-release-channels.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+date: 2023-06-16 10:30:05 -0530
+heading: Changing Meshery Release Channels
+title: Changing Meshery Release Channels
+author_contributor: Karan Thakur
```
[the remark this answers is not in the document]
Comment (Mohith234) [reply collections/_posts/2023-06-16-changing-meshery-release-channels.md:5]: @thisiskaransgit couldn't find the cause of the issue. I guess we need to change the author or @leecalcote could help with this on where this default comes from.

**Comment by Mohith234** [comment]
For now, I have removed the `author_contributor` field and by default, it shows Lee Calcote as an author.

Do review the other parts and let's close it 👍

// @thisiskaransgit

File: _sass/blog.scss [hunk 2]
```diff
@@ -55,7 +55,7 @@
 	}
 
 	.post-details {
-		border-bottom: 1px solid #eee;
+		border-bottom: 1px solid #00B39F;
```
Comment (thisiskaransgit) [inline _sass/blog.scss:58]:
```suggestion
		border-bottom: 1px solid var(--brand-color-secondary);
```

File: _sass/blog.scss [hunk 3]
```diff
@@ -97,16 +97,17 @@
 .blog-filter {
 	text-align: center;
 	a {
-		background: #eee;
+		background: #00B39F;
```
Comment (thisiskaransgit) [inline _sass/blog.scss:100]:
```suggestion
		background:  var(--brand-color-secondary);
```

File: _sass/blog.scss [hunk 4]
```diff
@@ -97,16 +97,17 @@
...
     padding: 3px 5px;
     font-size: .8em;
     border-radius: 5px;
-    color: #888;
+    color: #fff;
```
Comment (thisiskaransgit) [inline _sass/blog.scss:104]:
```suggestion
    color:  var(--color-white);
```

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Comment by samyakshah3008** [comment]
@Mohith234 Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

**Comment by Mohith234** [comment]
@samyakshah3008 I believe this PR has been completed and is ready to merge.

**Comment by thisiskaransgit** [comment]
@Chadha93, we're good to go with this PR

**Review by Chadha93 — APPROVED** [review]
Thank you, gents.

**Pull request merged**